modules/adding-tab-pods-page.adoc

```diff
@@ -58,3 +85,8 @@ export default function ExampleTab() {
 
 .Verification
 * Visit a *Pod* page to view the added tab.
+
+[NOTE]
+====
+You can check the list of dynamic plug-ins to be loaded in your browser in the *Console* tab, evaluate `SERVER_FLAGS.consolePlugins` to see the dynamic plug-in list passed from Bridge server to the Console frontend.
```

Comment:
If we add this, I would probably put this in a specific troubleshooting section. It feels a little out of place here and only needed if things go wrong.

If we do keep it, make it clear the Console tab is in the browser developer tools. I'd also remove references to "Bridge" since that's an internal name.